src/components/ui/Switch/switch.scss

```
@@ -0,0 +1,38 @@
+.Layer__Switch {
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+  font-size: var(--text-md);
+
+  [slot='indicator'] {
+    display: flex;
+    align-items: center;
+    height: 1.25rem;
+    width: 2.25rem;
+    border-radius: 1rem;
+    border: 1px solid var(--outline-default);
+    background: var(--bg-default);
+    transition: all 200ms;
+
+    &::before {
+      height: 1rem;
+      width: 1rem;
+      border-radius: 0.5rem;
+      background: var(--color-black);
+      content: '';
+      transition: all 200ms;
+    }
+  }
+
+  &[data-selected] {
+    [slot='indicator'] {
+      border-color: var(--color-black);
```

Let's use `--color-base-1000` for all `--color-black` and `--color-base-1000` for all `--color-white`.
```suggestion
      border-color: var(--color-base-1000);
```